javalanglib/src/main/scala/java/lang/_String.scala

```diff
@@ -332,9 +339,6 @@ object _String { // scalastyle:ignore
    *  and that we need to implement these very Scala methods.
    */
   private trait SpecialJSStringOps extends js.Any {
-    def length: Int
-    def charCodeAt(index: Int): Int
-
     def toLowerCase(): String
     def toUpperCase(): String
     def trim(): String
```

Comment:
I don't understand why the lower three methods (and the trait in general) are OK.

Reply:
Because they're not reachable. We only care about reachable stuff, since the Analyzer will eliminate everything else.

Reply:
Now, it's true that, for consistency, we could get rid of `SpecialJSStringOps` entirely. But if we do that, it would also make sense to get rid of all dependencies on `JSStringOps`, which would be quite annoying.